pyworkflow/__init__.py

```diff
@@ -148,6 +176,21 @@ class Config:
     SCIPION_DOMAIN = __get('SCIPION_DOMAIN', None)
     PW_ALT_TESTS_CMD = __get(PW_ALT_TESTS_CMD, getTestsScript())
```

Comment:
All other configuration variables start with SCIPION_ prefix, why not in this case? Moreover, can we try to reduce the parts in the name ( _ used)??

Reply:
You are right.....the name was more to avoid SCIPION in pyworkflow, but the rest are LEFT with SCIPION. I'll change it and try to find a better name with less words.

Reply:
Yes, probably PW_ would be a better prefix now, but I think we can live with SCIPION_ one and keep it for a bit more of backward compatibility.

Reply:
I've changesd it to SCIPION_LAUNCH_CMD

Reply:
Finally the name is SCIPION_TESTS_CMD